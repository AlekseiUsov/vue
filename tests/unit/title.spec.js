import { mount } from "@vue/test-utils";
import title from "../../src/components/atoms/ToDoTitle.vue";

describe("ToDoTitle.vue", () => {
  it("выводит корректный заголовок", () => {
    const wrapper = mount(title);
    expect(wrapper).toMatchSnapshot();
  });
});
