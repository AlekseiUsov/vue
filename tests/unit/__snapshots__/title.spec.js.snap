// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToDoTitle.vue выводит корректный заголовок 1`] = `<h1>to do list</h1>`;
